R/plot-etiology-regression.R

```diff
@@ -61,7 +61,7 @@
 #' @family visualization functions
 #' @export
 plot_etiology_regression <- function(DIR_NPLCM,stratum_bool,slice=1,plot_basis=FALSE,
-                                     truth=NULL,RES_NPLCM=NULL,do_plot=TRUE,return_metric=TRUE,
+                                     truth=NULL,RES_NPLCM=NULL,do_plot=TRUE,do_rug=FALSE, return_metric=TRUE,
```

Comment:
@realirena can we add do_rug in the the @param, so the function can recognize this argument.

Reply:
Just updated so do_rug is in @param